tools/epanet-output/src/errormanager.h

```diff
@@ -0,0 +1,25 @@
+/*
+ *  errormanager.h
+ *
+ *  Created on: Aug 25, 2017
+ *      Author: mtryby
+ */
+
+#ifndef ERRORMANAGER_H_
+#define ERRORMANAGER_H_
+
+#define ERR_MAXMSG 256
+
+typedef struct error_s {
+	int error_status;
+	void (*p_msg_lookup)(int, char*, int);
+} error_t;
+
+error_t* new_errormanager(void (*p_error_message)(int, char*, int));
+void dst_errormanager(error_t* error_handle);
+
+int throw_error(error_t* error_handle, int errorcode);
```

Comment:
i wonder about using the word `throw`, since C doesn't support exceptions, this word could be confusing. maybe `set_error` is more semantically accurate. I'm guessing that this is here to support the Python wrapper's exception model, but the code here is C, so it just feels weird.

Reply:
Yeah I agree `set_error` is more accurate. I will make that change. 

The SWIG wrapper actually gets generated in C and compiled into a python extension. You can see in `outputapi.i` starting at line 150 the custom exception block that get inserted into the wrapper code. Once you get more familiar with SWIG it won't seem as weird.